test/gtest/ucp/test_ucp_tag_fallback.cc

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+ *
```

Comment:
why need to create new file/class?
could just add new test to `test_ucp_tag`

Reply:
technically it is possible, but current test uses own infrastructure, including new NBX API which could not be reused from test_ucp_tag.

@yosefe what do you think?

Reply:
merged into existing file